config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Locations::Engine.routes.draw do
+  resources :floors
```

Comment:
I think you probably want to keep the same pattern for new/create as the other resources and do ```resources :floors, path_names: { new: "create" }```